local/index.js

```diff
@@ -1,4 +1,5 @@
 const tenantId = '$YOUR_TENANT_ID';
+const storeUrl = "https://www.example.com";
```

Comment:
I'm not sure if I like this being front and center at the top since its main use is for V1 blocks. @cameron-jones-volusion what do you think? I'm wondering if we update the line below to be `storeUrl: 'https://www.example.com',`

If we do keep this, the quotes should be updated to single instead of double.

Reply:
I think it's ok right here. @chris-decker-volusion Maybe we can add a comment explaining that this value is available if you need it, but not _required_ for anything.

@dallonf It might also be appropriate to change the value to `http://localhost:4000/`, since this value represents the URL that is currently being rendered—not necessarily the official domain of the merchant's site.